Update TBETA.ipynb

ernestamandine opened the pull request:
Fixed errors in scattering region, changed size of leads displayed.

Commit:
Update TBETA.ipynb

KrisCer commented on TBETA.ipynb:
```diff
@@ -506,7 +511,66 @@
+    "def plot_numbers_sys2():\n",
+    "        sites = list(sys2.sites()) #get the sites from kwant builder, the coordinates are shown in terms of lattice vectors\n",
+    "        fig, ax = plt.subplots() #only one subplot here\n",
+    "        ax.set_yticks([])\n",
```
Are all lines here really needed? For example,  line 523 - ax.set_yticks is definitely not needed if we don't actually display the plot, same with plotting the "numbers" in lines ~535-549. Probably only need to get sites, make a subplot, and then do kwant.plotter in order to get xaxis2,yaxis2, no?

KrisCer reviewed (changes requested):
Seems fine, except for the second plot_numbers function, where I think we can get rid of most of the lines.

KrisCer commented:
Please change the selection names from "Width" and "QD" to "Lead width" and "Junction width"

Commit:
Some Clean_up

Edited plot_numbers2 function, and changed slider names to "QD width" and "Lead width".
